pkg/controller/storagecluster/reconcile.go

```diff
@@ -651,7 +651,7 @@ func (r *ReconcileStorageCluster) ensureJobTemplates(sc *ocsv1.StorageCluster, r
 		}
 		osdCleanUpTemplate.Parameters = []openshiftv1.Parameter{
 			{
-				Name:     "FAILED_OSD_ID",
+				Name:     "FAILED_OSD_IDS",
```

Comment:
Why is this plural? What's the expected format? What happens if there's spaces?

Reply:
The FAILED_OSD_IDS would represent the removal of more than one osd at the same time.

Reply:
That only answers the first of three question.

Reply:
I think the format should include underscores. Since we are using `FAILED_OSD_IDS` as a parameter to the openshift template when we trigger the template to remove those osds - `oc process -n openshift-storage ocs-osd-removal -p FAILED_OSD_ID=${failed_osd_id} | oc create -f -`. `FAILED_OSD_IDS` will be replaced by the osd ids- 0,1 as such. I am not sure what will happen if there are spaces instead of underscores though, haven't tested that way.

Reply:
This seems exceedingly error prone. You could easily create a list like `FAILED_OSD_IDS="1 2 3"` which would be a valid parameter as far as `oc process` is concerned. There should be a well-documented format (my preferences would be for commas), and validation that if anything other than numbers or the divisor are detected that the job fail (ideally the processing should fail as well, but I don't think we can do parameter validation in templates).

Reply:
To clarify, the Rook implementation expects a comma-separated list of numerical IDs. Rook will iterate over each of the IDs so that a single job can trigger the removal of multiple OSDs if desired. If not in the expected comma-separated format, or an ID cannot be converted to an int, or if an OSD ID is not found, the invalid ID(s) will be skipped and an error will be logged in the job. For example, if you use spaces, you'll just get errors in the log and no OSDs would be removed.

Reply:
This information should be captured in the description of the parameter. I believe there's a Parameter.Description field: https://docs.openshift.com/container-platform/4.5/rest_api/template_apis/template-template-openshift-io-v1.html#template-template-openshift-io-v1

While we're at it, a more readable `DisplayName` like "OSD IDs" would probably be useful.

Reply:
Okay. Noted. Two things to add: description and displayName of the parameter.

Reply:
@jarrpa Could you please review? I added a description and a DisplayName for the parameter.